src/core/lib/surface/server.h

```diff
@@ -66,6 +69,10 @@
 #include "src/core/lib/transport/metadata_batch.h"
 #include "src/core/lib/transport/transport.h"
 
+#define GRPC_ARG_SERVER_MAX_PENDING_REQUESTS "grpc.server.max_pending_requests"
```

Comment:
Shouldn't these be defined in the public header file?

Reply:
holding them private for the moment: we need the escape hatch for rollout, but I think ultimately we want a different knob.